configs/EMAX_F405_ELRS_AIO/config.h

```diff
@@ -91,6 +91,4 @@
 #define FLASH_SPI_INSTANCE SPI2
 #define GYRO_1_SPI_INSTANCE SPI1
 #define GYRO_1_ALIGN CW90_DEG
-#define GYRO_1_ALIGN_YAW 900
-#define GYRO_1_SENSOR_ALIGN CW0_DEG

```

Comment:
src/main/cli/settings.c:1745:    { "gyro_1_sensor_align", VAR_UINT8  | HARDWARE_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_ALIGNMENT }, PG_GYRO_DEVICE_CONFIG, PG_ARRAY_ELEMENT_OFFSET(gyroDeviceConfig_t, 0, alignment) },

Reply:
Yes, but it is base on `GYRO_1_ALIGN`

Reply:
i don't need it to say `_SENSOR_`, but such is personal opinion

Reply:
For follow up another improvement would be using IMU alignment as in

```bash
src/main/sensors/compass.c:225:            *alignment = MAG_HMC5883_ALIGN;
src/main/sensors/compass.c:241:            *alignment = MAG_LIS2MDL_ALIGN;
src/main/sensors/compass.c:257:            *alignment = MAG_LIS3MDL_ALIGN;
src/main/sensors/compass.c:273:            *alignment = MAG_AK8975_ALIGN;
src/main/sensors/compass.c:294:            *alignment = MAG_AK8963_ALIGN;
src/main/sensors/compass.c:310:            *alignment = MAG_QMC5883L_ALIGN;
src/main/sensors/compass.c:326:            *alignment = MAG_IST8310_ALIGN;
```

This way multiple IMU's can be supported with [custom] alignments in a single target. Now we have to duplicate targets when orientation is different.

Reply:
@haslinghuis : IMU alignment is used-configurable, target define is only setting default value on reset. So it is almost impossible to use per-imu parameter for target (it would need to autodetect IMU before resetting PG. IMO not worth the effort).

Reply:
@ledvinap Yes you are right - PG reset won't know used gyro hardware.

Think this PR is ready to merge or should we wait for https://github.com/betaflight/betaflight/pull/14092 being merged ?

Reply:
This one should go first

Reply:
Okay let's merge this.